tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -50,13 +51,13 @@ object VarDictJava extends Configuration {
   val RootDirConfigKey = "vardictjava.dir"
 
   /** Path to the VarDictJava directory. */
-  lazy val RootDir = configure[Path](RootDirConfigKey)
+  val RootDir = configure[Path](RootDirConfigKey)
```

Comment:
Yup - it should only matter if a pipeline references the object, or constructs an instance of the class.